### PR TITLE
local storage implemented

### PR DESCRIPTION
**Pull request by Yonasle**
-Professional Readme file added
-Linter Configured
-Local Storage implemented
-Best practice HTML, CSS and Javascript applied

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -243,3 +243,29 @@ emailInput.addEventListener('input', () => {
...
+messageField.addEventListener('input', saveFormData);
+
+window.addEventListener('load', () => {
+  const storedData = JSON.parse(localStorage.getItem('formData'));
+  if (storedData) {
+    usernameField.value = storedData.username;
+    emailField.value = storedData.email;
+    messageField.value = storedData.message;
+  }
+});
```
Comment (iambenkis) [inline script.js:271]: - [ ] Keep doing the great work 👍

**Review by iambenkis — COMMENTED** [review]
Hi Team , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted a project that should contain contribution of your partner because it is a group project, kindly make sure this pull request contains contribution of all the team members 👍  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@iambenkis` in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Yonasle** [comment]
Microverse
 Please Refer the following image for review

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @Yonasle 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**